### ISSUE TITLE
ModuleNotFoundError

### ISSUE THREAD

**Issue by wooddoor** [aroberge/easygui_qt#27]
File "D:\Program Files\Python36\lib\site-packages\easygui_qt\__init__.py", line 6, in <module>
    from .easygui_qt import *
  File "D:\Program Files\Python36\lib\site-packages\easygui_qt\easygui_qt.py", line 32, in <module>
    import language_selector
ModuleNotFoundError: No module named 'language_selector'

  File "D:\Program Files\Python36\lib\site-packages\easygui_qt\utils.py", line 125, in <module>
    class MyPageDialog(QtGui.QDialog):
AttributeError: module 'PyQt5.QtGui' has no attribute 'QDialog'

**Comment by aroberge** [comment]
Thank you for reporting this bug. Unfortunately, I do not have much time to look at it right now - but I will try to do my best.

**Comment by wooddoor** [comment]
I'm a beginner in Python, and I used to give up learning because of the Python GUI. Now there is a Python GUI library that is so easy to use, and hopefully it will get better and better thanks to the author's hard work.
My English is very poor, above for machine translation.

**Comment by aroberge** [comment]
I have uploaded the new version of easygui_qt to pypi. If this is from where you had installed it, you will need to reinstall it again.

**Comment by Deansenecal** [comment]
ModuleNotFoundError: No module named 'easygui_qt'.  I have imported easygui_qt.   When running my script in pycharm, everything works perfectly.   When I attempt to create an MAC exe file with pyinstaller everything complies correctly but when I try to run the exe file I get the following error.   ModuleNotFoundError: No module named 'easygui_qt'.   I have attempted to include the easygui_qt path iinto my enviroment.  No joy.   I have attempted to include the file location in my pyinstaller instructions, no joy.   I am using Python 3.9 on iMAC running Monterey 12.6.3
